sys/net/gnrc/network_layer/sixlowpan/gnrc_sixlowpan.c

```diff
@@ -67,19 +67,18 @@ void gnrc_sixlowpan_dispatch_recv(gnrc_pktsnip_t *pkt, void *context,
 
     (void)context;
     (void)page;
-#ifdef MODULE_CCNLITE
+#ifdef MODULE_CCN_LITE
```

Comment:
The code in here doesn't look CCN-lite related. How about

```suggestion
#ifndef MODULE_GNRC_IPV6
```

(or using `#ifdef MODULE_GNRC_IPV6` and turn the branches around.